package.json

```diff
@@ -45,13 +45,11 @@
     "broadcastchannel-polyfill": "^1.0.0",
     "dedent": "^0.7.0",
     "history": "^4.9.0",
-    "marksy": "^8.0.0",
     "mdast-builder": "^1.1.1",
     "mdast-util-assert": "^3.0.0",
     "mdast-zone": "^4.0.0",
     "merge-anything": "^3.0.3",
     "mousetrap": "^1.6.5",
-    "normalize-newline": "^3.0.0",
```

Comment:
Did not find any usages of this lib in the codebase.  Does it seem safe to remove?  (Removing dependencies is hard without full context!)

Reply:
Looks like this lib was used for an old Deck component.
So safe to delete too!